StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Saved Payment Method Screen/Vertical Saved Payment Method Screen/VerticalSavedPaymentMethodsViewController.swift

```diff
@@ -206,6 +206,16 @@ class VerticalSavedPaymentMethodsViewController: UIViewController {
         minHeightConstraint.isActive = true
     }
 
+    private func navigationBarStyle() -> SheetNavigationBar.Style {
+        if let bottomSheet = self.bottomSheetController,
+           bottomSheet.contentStack.first is VerticalSavedPaymentMethodsViewController,
```

Comment:
Interesting, ooc why do we need this check?

Reply:
AH! Not needed!  I must have added it for safety, but is not necessary. thank you!